src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -121,6 +121,11 @@ public class TraceLoadedDotNetRuntime
         /// Returns the .NET startup flags
         /// </summary>
         public StartupFlags StartupFlags { get; internal set; }
+        /// <summary>
+        /// Date and time of when the runtime was built
+        /// This is useful when a more detailed version is not present
+        /// </summary>
+        public DateTime RuntimeBuiltTime { get; internal set; }
```

Comment:
❓ Is there a reason not to use `DateTime?` in order to clean up the code and keep it clear that this may or may not have a value?

❓ Is there a reason not to use `DateTimeOffset` instead of `DateTime`? It is a generally more reliable type for representing points in time in an application.